src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -214,8 +214,116 @@ As AWS is a paying service, when the Scheduler is stopped normally (without remo
+ If specified, this parameter has to refer to an existing subnet in the region affected to the specified VPC, and has to comply with the subnet ID format.
+ This parameter has to be filed only if the *defaultVpcId* is also completed.
+ Otherwise, this parameter has to be left blank to trigger networking autoconfiguration.
+
+	- *defaultSecurityGroup:* This parameter receives the ID of the security group to spawn instances into.
+ If this parameter does not meet the requirement regarding the providing the provided VPC and subnet, a new security group will be generated.
+ This parameter is mandatory, and has to comply with the format of the ID of the AWS security groups.
+
+	- *region:* The administrator specifies here the AWS region to allocate the cluster into.
+ This parameter is mandatory and has to be valued after the name of an AWS region.
+ Please see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions[the related documentation] to see the available region names.
+
+	- *rmUrl:* This field receives the URL to access the resource manager from the nodes.
+ The URL must comply with the specification of the communication protocols used by ProActives, and can therefore be prefixed with *pnp://*, *pnps://* or *pamr://*.
+ This parameter is mandatory.
+
+	- *rmCredential:* The administrator has to provide this field with the content of the credentials file created from the resource manager.
+ This parameter is mandatory.
+
+	- *rmHostname:* This field is to be filled with the domain name or the IP of the host operating the resource manager.
+ This option is mandatory.
+
+	- *externalStartupScript:* The administrator has to provide a script to configure AWS instances to work with ProActive.
+ Usually, if not provided by the AMI, this script is expected to download the Java Runtime Environment and ProActive node.jar agent file.
+ This field is expected to contain the content of the script, and start with a Shebang, as an AWS imposes it.
+ If left blank, the script is automatically generated for the Linux OS.
+
+	- *maxNodePerVM:* The administrator specifies the amount of nodes to be deployed on each AWS instance.
+ This parameter is mandatory, and has to be an integer equal or greater than one.
+
++
+WARNING: Please ensure this parameter is aligned with with the capacity of the specified instance type mentionned in the *defaultInstanceType* field.
+
+	- *deploymentTimeOut:* This field contains the delay in seconds for a node to be deployed and to contact back the Resource Manager before being declared as lost.
+ This parameter is mandatory.
+
+	- *cleanDelay:*	The administrator can define the periodicity in seconds for unused instance removal.
+ This parameter is mandatory.
+
+Using this configuration, you can start a Resource Manager and a Scheduler using the */bin/proactive-server* script.
```

Comment:
Probably add a link to:
https://www.activeeon.com/public_content/documentation/latest/admin/ProActiveAdminGuide.html#_run_the_proactive_scheduler

Reply:
Ok.